packages/cli/lib/LocalDevManager.js

```diff
@@ -1,5 +1,6 @@
 const chokidar = require('chokidar');
```

Comment:
Keeping this file around makes it easy to maintain the existing functionality while we work to build out the v2 behavior that will eventually replace this. We fallback to using this version of the dev manager when the user does not pass in the `--extension` option.